src/core_modules/capture-core/components/DataEntries/withErrorMessagePostProcessor/UniqueTEADuplicate/ExistingTEILoader.types.ts

```diff
@@ -0,0 +1,18 @@
+import type { ErrorData } from '../../../D2Form/FormBuilder';
```

Comment:
Delete the src/core_modules/capture-core/components/DataEntries/withErrorMessagePostProcessor/UniqueTEADuplicate/ExistingTEILoader.types.ts file, it is not being used